starfish/pipeline/features/spots/detector/gaussian.py

```diff
@@ -170,3 +125,77 @@ def add_arguments(cls, group_parser):
             "--overlap", default=0.5, type=float, help="dots with overlap of greater than this fraction are combined")
         group_parser.add_argument(
             "--show", default=False, action='store_true', help="display results visually")
+
+
+# TODO ambrosejcarr: make this return IntensityTable instead of SpotAttributes
```

Comment:
Saving this TODO for future PRs. 

The below function is executed by `detect_spots`.